### ISSUE TITLE
Clean branches and fix build error on master

### ISSUE THREAD

**Issue by rdeltour** [daisy/pipeline-mod-braille#22]
The `master` branch is not a child of the `release/v1.9` tag, we need to clean this up, maybe just merge the branch containing the release to the master.

Also, when building the `master` branch, I get an error:

```
[INFO] ------------------------------------------------------------------------
[INFO] Building DAISY Pipeline 2 Braille Module :: liblouis-core 3.0.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[WARNING] The POM for org.liblouis:liblouis-java:jar:1.3.0 is missing, no dependency information available
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] DAISY Pipeline 2 :: Braille Modules Bill Of Materials SUCCESS [  0.123 s]
[INFO] DAISY Pipeline 2 :: Braille Modules Parent POM ..... SUCCESS [  0.059 s]
[INFO] DAISY Pipeline 2 Braille Module :: common-utils .... SUCCESS [  9.457 s]
[INFO] DAISY Pipeline 2 Braille Module :: css-core ........ SUCCESS [  7.339 s]
[INFO] DAISY Pipeline 2 Braille Module :: css-calabash .... SUCCESS [ 23.015 s]
[INFO] DAISY Pipeline 2 Braille Module :: css-utils ....... SUCCESS [ 17.984 s]
[INFO] DAISY Pipeline 2 Braille Module :: pef-calabash .... SUCCESS [  0.402 s]
[INFO] DAISY Pipeline 2 Braille Module :: pef-utils ....... SUCCESS [  0.222 s]
[INFO] DAISY Pipeline 2 Braille Module :: obfl-utils ...... SUCCESS [  0.181 s]
[INFO] DAISY Pipeline 2 Braille Module :: liblouis-native . SUCCESS [  0.430 s]
[INFO] DAISY Pipeline 2 Braille Module :: liblouis-core ... FAILURE [  0.029 s]
[INFO] DAISY Pipeline 2 Braille Module :: liblouis-saxon .. SKIPPED
(...)
[ERROR] Failed to execute goal on project liblouis-core: Could not resolve dependencies for project org.daisy.pipeline.modules.braille:liblouis-core:bundle:3.0.0-SNAPSHOT: Failure to find org.liblouis:liblouis-java:jar:1.3.0 in http://download.eclipse.org/rt/eclipselink/maven.repo was cached in the local repository, resolution will not be reattempted until the update interval of eclipselink has elapsed or updates are forced -> [Help 1]
```

It seems Maven Central contains `org.liblouis:liblouis-jar:1.2.0` but not `1.3.0`. @bertfrees can you look into it?

**Comment by bertfrees** [comment]
Strange. liblouis-java 1.3.0 is not in central yet but it is staged, and it worked on Travis.

Yes, master is not a child of v1.9. I thought this wouldn't be a problem because you have the same situations when you do release branches. Yes, merging the release branch to master is what I had in mind too (if you want to delete the release branch and be sure it is not garbage collected).

**Comment by rdeltour** [comment]
> liblouis-java 1.3.0 is not in central yet, but it is staged, and it worked fine on Travis.

Oh, that's it then. The staging repo is enabled on Travis, but may not be on users' setup (e.g. it's not enabled on mine at the moment).
It's a bit dangerous that Travis enables repos that users may not have, but otherwise we would have to release to Maven Central sequentially, not many staged projects as a bulk, otherwise Travis wouldn't find the yet-unreleased-to-central projects.

**Comment by bertfrees** [comment]
I have released liblouis-java 1.3.0. You want me to do the 1.9 merge?
